### PR TITLE
Tooling setup

### PR DESCRIPTION
**Pull request by lvdoorn**
I have changed most files formatting and fixed all JSHint errors. I assume there will be a lot of merge conflicts though.

### CODE REVIEW

File: src/modules/scatter.js [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
         parentSelector: "#bodyrow",
         body: function () {
-        var w = 720,
-            h = 350,
-            pad = 30,
-            left_pad = 40,
-            data1 = [10, 20, 30, 40, 50, 60, 70, 80],
-            data2 = [30, 60, 90, 120, 150, 180, 210, 240];
+            var w = 720,
+                h = 350,
```
Comment (DaanvanderValk) [inline src/modules/scatter.js]: Was this alignment preferred according to LintJS?

File: src/settings.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+
 function Settings() {
-	this.protocol = 'http';
-	this.hostname = '146.185.128.124';
-	this.port = '';
-	this.host = this.protocol + '://' + this.hostname + ((this.port.length > 0) ? ':' + this.port : '') + '/api';
+    this.protocol = 'http';
+    this.hostname = '146.185.128.124';
+    this.port = '';
+    this.host = this.protocol + '://' + this.hostname + ((this.port.length > 0) ? ':' + this.port : '') + '/api';
```
Comment (DaanvanderValk) [inline src/settings.js]: So shortcut if-else statements (like this one) are allowed, but an usual if-statement should always contain parentheses?

**Comment by DaanvanderValk** [comment]
Looks nice: despite the large amount of commits/changed files, the changes are pretty self-explanatory.

File: src/modules/scatter.js (hunk 1 above)
Comment (agudek) [inline src/modules/scatter.js]: I can see why that would be desired. It is inside the function { ... } block from line 8, so needs 1 more tab.

File: src/settings.js (hunk 2 above)
Comment (mboom) [inline src/settings.js]: @DaanvanderValk yes indeed. the if, then and else part are clearly separated by ? and :. Also the statement is enclosed in parenthesis, so there is no ambiguity.

**Pull request merged**